site/gatsby-site/migrations/2024.02.01T20.03.56.set-incidents-submissions-reports-created_at.js

```diff
@@ -0,0 +1,180 @@
+const config = require('../config');
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  // New created_at field on incidents and submissions collections
```

Comment:
Please change the comment to include all 6 collections included in this migration